downstream/modules/platform/proc-creating-the-custom-execution-environment-definition.adoc

```diff
@@ -0,0 +1,161 @@
+
+[role="_abstract"]
+
+Once the {Builder} RPM is installed, use the following steps to create your custom {ExecEnvShort}.
+
+. Create a directory for the build artifacts used when creating your custom {ExecEnvShort}.	Any new files created with the steps below will be created under this directory.
++
+----
+$ mkdir $HOME/custom-ee $HOME/custom-ee/files
+$ cd $HOME/custom-ee/
+----
++
+
+. Create an `execution-environment.yml` file that defines the requirements for your custom {ExecEnvShort}. 
+
+.. Override the base image to point to the minimal execution environment available in your {PrivateHubName}.
+
+.. Define the additional build files needed to point to any disconnected content sources that will be used in the build process.
+Your custom `execution-environment.yml` file should look similar to the following example:
+
++
+----
+$ cat execution-environment.yml
+---
+version: 3
+
+images:
+  base_image:
+    name: private-hub.example.com/ee-minimal-rhel8:latest
+
+dependencies:
+  python: requirements.txt
+  galaxy: requirements.yml
+
+additional_build_files:
+  - src: files/ansible.cfg
+    dest: configs
+  - src: files/pip.conf
+    dest: configs
+  - src: files/hub-ca.crt
+    dest: configs
+  # uncomment if custom RPM repositories are required
+  #- src: files/custom.repo
+  #  dest: configs
+
+additional_build_steps:
+  prepend_base:
+    # copy a custom pip.conf to override the location of the PyPI content
+    - ADD _build/configs/pip.conf /etc/pip.conf
+    # remove the default UBI repository definition
+    - RUN rm -f /etc/yum.repos.d/ubi.repo
+    # copy the hub CA certificate and update the trust store
+    - ADD _build/configs/hub-ca.crt /etc/pki/ca-trust/source/anchors
+    - RUN update-ca-trust
+    # if needed, uncomment to add a custom RPM repository configuration
+    #- ADD _build/configs/custom.repo /etc/yum.repos.d/custom.repo
+
+  prepend_galaxy:
+    - ADD _build/configs/ansible.cfg ~/.ansible.cfg
+
+...
+----
++
+
+. Create an `ansible.cfg` file under the `files/` subdirectory that points to your {PrivateHubName}, and contains the token used to authenticate to the hub and download collections.
++
+----
+$ cat files/ansible.cfg
+[galaxy]
+server_list = private_hub
+
+[galaxy_server.private_hub]
+url = https://private-hub.example.com/api/galaxy/
+----
++
+. Create a `pip.conf` file under the `files/` subdirectory which points to the internal PyPI mirror (a web server or something like Nexus):
++
+----
+$ cat files/pip.conf
+[global]
+index-url = https://<pypi_mirror_fqdn>/
+trusted-host = <pypi_mirror_fqdn>
+----
++
+
+* OPTIONAL: If a `bindep.txt` file is being used to add RPMs the custom {ExecEnvShort}, create a `custom.repo` file under the `files/` subdirectory that points to your disconnected Satellite or other location hosting the RPM repositories.  If this step is necessary, uncomment the steps in the example `execution-environment.yml` file that correspond with the `custom.repo` file.
```

Comment:
this should be bullet list 5 but optional, not a sub-bullet under list item 4.  i think that means the line should begin with `.` instead of `*`?